### ISSUE TITLE
Hotkey to go back to their box?

### ISSUE THREAD

**Issue by aawiseman** [hotosm/osm-tasking-manager2#808]
Sometimes when you're editing you bump your mouse or something and you pan really far from where your task box is. Would be nice to have a way to go back to your box without saving, closing the window, and re-opening from the Tasking Manager. I've seen this happen to newbies a lot, and it happens to me sometimes too, and I'm to exactly sure how/why. I wonder if it's a bug? Or just an unlucky click.

**Comment by pellis2** [comment]
I've experienced this as well. Have you ever experienced it in ID Editor outside of a HOT Tasking Manager session? I have not; I've only ever had it happen to me in a HOT Tasking Manger editing session using ID Editor. Curious if that might help the developers at all.

**Comment by aawiseman** [comment]
@pellis2 that's true, it's only been in the Tasking Manager -- then again I mostly just use the Tasking Manager
